components/admin/requests/reports/GenerateModal.tsx

```diff
@@ -79,6 +79,13 @@ export default function GenerateReportModal(props: Props) {
   >(GENERATE_APPLICATIONS_REPORT_QUERY, {
     onCompleted: data => {
       if (data.generateApplicationsReport.ok) {
+        const link = document.createElement('a');
+        link.setAttribute('href', data.generateApplicationsReport.link as string);
+        link.style.visibility = 'hidden';
+        document.body.appendChild(link);
+        link.click();
+        document.body.removeChild(link);
```

Comment:
heard on co-op that this is better than window.open() bc it won't trigger browser pop-up blocker. I'm not FE expert so open to different ideas